tests/application/test_base_dto.py

```diff
@@ -0,0 +1,28 @@
+from erica.application.base_dto import CamelCaseModel
+
+
+class CamelCaseModelChild(CamelCaseModel):
+    name_of_pet: str
+
+
+class TestCamelCaseModel:
+
+    def test_if_parsed_from_dict_with_snake_case_attribute_then_transform_to_object_with_snake_case_attributes(self):
+        name_of_pet = 'Pickett'
+        resulting_object = CamelCaseModelChild.parse_obj({'name_of_pet': name_of_pet})
+
+        assert resulting_object.name_of_pet == name_of_pet
+
+    def test_if_parsed_from_dict_with_camel_case_attribute_then_transform_to_object_with_snake_case_attributes(self):
+        name_of_pet = 'Pickett'
+        resulting_object = CamelCaseModelChild.parse_obj({'nameOfPet': name_of_pet})
+
+        assert resulting_object.name_of_pet == name_of_pet
```

Comment:
Just to be very sure: Can we add a test for something like person_a_name (with only one letter)? Just thinking about that pyhumps problem that we once had.